ckanext/budgetviz/groupbarchart_munc/theme/public/groupbarchart-main.js

```diff
@@ -0,0 +1,91 @@
+var addMiscElements = function() {
+    d3.select("#viz-header").text(("BBMP Budget Summary Statement"))
+}
+
+var populateSelection = function(data) {
+    var select = d3.select("#select-list")
+        .append("form")
+        .attr("role", "form")
+        .selectAll("div")
+        .data(data)
+        .enter().append("div")
+        .attr("class", "radio")
+        .on("click", function(d) {
+            drawchart(d)
+        })
+
+
+    var inner_div_select = select.append("div")
+        .attr("class", "inner-div")
+
+    inner_div_select
+        .append("label")
+        .attr("value", function(d) {
+            return d.name;
+        })
+        .attr("for", function(d) {
+            return d.name;
+        })
+        .text(function(d) {
+            return d.name;
+        })
+        .style("margin", "0px 0px 0px 0px");
+
+
+    inner_div_select
+        .insert("input", ":first-child")
+
+    .attr({
+            "name": "radio-button",
+            "type": "radio"
+        })
+        .attr("value", function(d) {
+            return d.name;
+        })
+        .attr("id", function(d) {
+            return d.name;
+        })
+
+
+    d3.select("#select-list").select("form").select("div").select("input").attr("checked", "")
+
+
+}
+
+
+
+function drawchart(data) {
+
+
+    nv.addGraph(function() {
+        var chart = nv.models.multiBarChart()
+            .color(["#002A4A", "#FF9311", "#D64700", "#17607D"])
+            .x(function(d) {
```

Comment:
Can we declare color pallets as class variables at the top so that it's easier to modify whenever we need?